conf/messages.en

```diff
@@ -7,6 +7,8 @@ site.change = Change
 site.confirm = Confirm
 site.yes = Yes
 site.no = No
+site.true = true
```

Comment:
I don't think this needs to be in messages file since it is not displayed to user